ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -46,8 +46,7 @@ cp \
     -t "${SHARED_DIR}" \
     "${dir}/auth/kubeconfig" \
     "${dir}/metadata.json" \
-    "${dir}/terraform.tfstate" \
-    "${dir}/.openshift_install.log"
+    "${dir}/terraform.tfstate"
```

Comment:
Not clear to me why we need this one either.

Reply:
From [here][1].  Possibly a relic of SSH-based gathers, dropped in #6336.

[1]: https://github.com/openshift/release/pull/6708/commits/de3de20d02e1a596daeaeae58b9fd6858c8b5353#diff-aa102aaad663ef68f918acd81caa3cfeR48

Reply:
It was definitely necessary when it was added.